go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/rclone/rclone
 
-go 1.19
```

Comment:
Please don't change the go version! It will break a huge amount of stuff.

Reply:
Won't do! I will make sure this is removed on the final version!

Reply:
I am having issues trying to just update the dependencies by running `go get -u github.com/henrybear327/Proton-API-Bridge@master`. 

This part will be automatically added to the `go.mod` file
```
go 1.21

toolchain go1.21.1
```. 

Trying to resolve this issue now :(

Reply:
Resolved by 
- `brew install go@1.19`
- manually removing the dependencies `Proton-API-Bridge` and `proton-go-api` from both `go.mod` and `go.sum` files
- run `go get github.com/henrybear327/Proton-API-Bridge@master` to reinstall the package

Reply:
Don't use `-u` with `go get` - it is almost always the wrong thing to do! It is a leftover from the early days of go.

Reply:
Thanks! Will put it on my notes :(